implementations/test/ERC725.test.ts

```diff
@@ -64,6 +64,22 @@ describe('ERC725', () => {
 
         expect(await ethers.provider.getBalance(contract.address)).to.equal(deployParams.funding);
       });
+
+      it('should have registered the ERC725X & ERC725Y interfaces', async () => {
+        const accounts = await ethers.getSigners();
+
+        const deployParams = {
+          newOwner: accounts[0].address,
+          funding: ethers.utils.parseEther('10'),
+        };
+
+        const contract = await new ERC725__factory(accounts[0]).deploy(deployParams.newOwner, {
+          value: deployParams.funding,
+        });
+
+        expect(await contract.supportsInterface(INTERFACE_ID.ERC725X)).to.be.true;
+        expect(await contract.supportsInterface(INTERFACE_ID.ERC725X)).to.be.true;
```

Comment:
You have a duplicate. I assume you meant to test ERC725Y here: 
```suggestion
        expect(await contract.supportsInterface(INTERFACE_ID.ERC725Y)).to.be.true;
```

Reply:
Eagle eyes

Reply:
Indeed very good catch! Fixing in #264